shortstories/views.py

```diff
@@ -9,3 +10,16 @@ def index(request):
 def detail(request, shortstory_id):
     shortstory =  get_object_or_404(ShortStory, pk=shortstory_id)
     return render(request,'shortstories/detail.html',{'shortstory':shortstory})
+
+def post_new(request):
```

Comment:
let's rename the view name and its url to `create`

Reply:
done.